### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
some improvements for models documentation aimed to increase visibility of public models available through Open Model Zoo

- [x] clarify pre-trained model term (it is Intel pre-trained models)

- [x] add colorization models to public models index

- [x] change links for public models from frameworks to model name

- [x] fill public models general info (reference accuracy values and theoretical complexity)

### CODE REVIEW

File: models/public/index.md [hunk 1]
```diff
@@ -209,6 +209,15 @@ The task of action recognition is to predict action that is being performed on a
 | ------------------------- | ---------------| -------------- | ------ | ------- |
 | RGB-I3D, pretrained on ImageNet\* | [TensorFlow\*](./i3d-rgb-tf/i3d-rgb-tf.md) | i3d-rgb-tf | | |
 
+## Colorization
+
+Colorization task is to predict colors of scane from grayscale image.
```
Comment (Wovchena) [inline models/public/index.md:214]:
```suggestion
Colorization task is to predict colors of scene from grayscale image.
```

Comment (IRDonch) [inline models/public/index.md:214]: > scane

scene?

Comment (vladimir-dudnik) [reply models/public/index.md]: yes, typo fixed by Anna's commit later

Comment (vladimir-dudnik) [reply models/public/index.md]: yes, typo fixed by Anna commit later

File: models/intel/index.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Overview of OpenVINO&trade; Toolkit Pre-Trained Models
+# Overview of OpenVINO&trade; Toolkit Intel Pre-Trained Models
```
[the remark this answers is not in the document]
Comment (vladimir-dudnik) [reply models/intel/index.md]: fixed

File: models/public/googlenet-v1/googlenet-v1.md [hunk 3]
```diff
@@ -21,6 +21,11 @@ The model output for `googlenet-v1` is the typical object classifier output for
 
 ## Accuracy
 
+| Metric | Value |
+| ------ | ----- |
+| Top 1  | 68.928|
+| Top 5  | 89.144|
+
 See [https://github.com/BVLC/caffe/tree/master/models/bvlc_googlenet](https://github.com/BVLC/caffe/tree/master/models/bvlc_googlenet).
```
Comment (eaidova) [inline models/public/googlenet-v1/googlenet-v1.md:29]: just for alignment with other models
```suggestion
See [original repository](https://github.com/BVLC/caffe/tree/master/models/bvlc_googlenet).
```

File: models/public/index.md [hunk 4]
```diff
@@ -9,64 +9,64 @@ The models can be downloaded via Model Downloader
...
+| MobileNet V2 1.0 224        | Caffe\*<br>TensorFlow\*<br>PyTorch\*| [mobilenet-v2](./mobilenet-v2/mobilenet-v2.md) <br>[mobilenet-v2-1.0-224](./mobilenet-v2-1.0-224/mobilenet-v2-1.0-224.md)<br>[mobilenet-v2-pytorch](./mobilenet-v2-pytorch/mobilenet-v2-pytorch.md) | 71.218/90.178<br>70.20/89.61<br>71.81/90.396 | 0.615~0.876 | 3.489 |
+| MobileNet V2 1.4 224        | TensorFlow\*                       | [mobilenet-v2-1.4-224](./mobilenet-v2-1.4-224/mobilenet-v2-1.4-224.md) | 74.09/91.97 | 1.183  | 6.087 |
+| MobileNet V3 Small 1.0      | TensorFlow\*                       | [mobilenet-v3-small-1.0-224-tf](./mobilenet-v3-small-1.0-224-tf/mobilenet-v3-small-1.0-224-tf.md) | 67.36/87.45 | 0.121 | 2.537 |
+| MobileNet V3 Large 1.0      | TensorFlow\*                       | [mobilenet-v3-large-1.0-224-tf](./mobilenet-v3-large-1.0-224-tf/mobilenet-v3-large-1.0-224-tf.md) | 75.70/92.76 | 0.4536 | 5.4721 |
+| ResNet 34                   | PyTorch\*                          | [resnet-34-pytorch](./resnet-34-pytorch/resnet-34-pytorch.md) | 73.30/91.42 | 7.3409 | 21.7892 |
```
Comment (eaidova) [inline models/public/index.md:41]: looks like ResNet 18 missed in index.md

File: models/public/index.md [hunk 5]
```diff
@@ -9,64 +9,64 @@ The models can be downloaded via Model Downloader
...
-| ResNet 50, alpha=0.125    | [MXNet\*](./octave-resnet-50-0.125/octave-resnet-50-0.125.md)         | octave-resnet-50-0.125    | | 7.221 | 25.551 |
-| ResNet 101, alpha=0.125   | [MXNet\*](./octave-resnet-101-0.125/octave-resnet-101-0.125.md)       | octave-resnet-101-0.125   | | 13.387 | 44.543 |
-| ResNet 200, alpha=0.125   | [MXNet\*](./octave-resnet-200-0.125/octave-resnet-200-0.125.md)       | octave-resnet-200-0.125   | | 25.407 | 64.667 |
-| ResNeXt 50, alpha=0.25    | [MXNet\*](./octave-resnext-50-0.25/octave-resnext-50-0.25.md)         | octave-resnext-50-0.25    | | 6.444 | 25.02 |
-| ResNeXt 101, alpha=0.25   | [MXNet\*](./octave-resnext-101-0.25/octave-resnext-101-0.25.md)       | octave-resnext-101-0.25   | | 11.521 | 44.169 |
-| SE-ResNet 50, alpha=0.125 | [MXNet\*](./octave-se-resnet-50-0.125/octave-se-resnet-50-0.125.md)   | octave-se-resnet-50-0.125 | | 7.246 | 28.082 |
+| DenseNet 121, alpha=0.125 | MXNet\*        | [octave-densenet-121-0.125](./octave-densenet-121-0.125/octave-densenet-121-0.125.md) | 76.066/93.044 | 4.883 | 7.977 |
```
Comment (eaidova) [inline models/public/index.md:62]: if I right remember, we discussed merging OctaveNets to classification models list

File: models/public/index.md [hunk 6]
```diff
@@ -9,64 +9,64 @@ The models can be downloaded via Model Downloader
...
+| MobileNet V1 1.0 224        | Caffe\*<br>TensorFlow\*            | [mobilenet-v1-1.0-224](./mobilenet-v1-1.0-224/mobilenet-v1-1.0-224.md)<br>[mobilenet-v1-1.0-224-tf](./mobilenet-v1-1.0-224-tf/mobilenet-v1-1.0-224-tf.md)| 69.496/89.224<br>71.03/89.94 | 1.148 | 4.221 |
+| MobileNet V2 1.0 224        | Caffe\*<br>TensorFlow\*<br>PyTorch\*| [mobilenet-v2](./mobilenet-v2/mobilenet-v2.md) <br>[mobilenet-v2-1.0-224](./mobilenet-v2-1.0-224/mobilenet-v2-1.0-224.md)<br>[mobilenet-v2-pytorch](./mobilenet-v2-pytorch/mobilenet-v2-pytorch.md) | 71.218/90.178<br>70.20/89.61<br>71.81/90.396 | 0.615~0.876 | 3.489 |
+| MobileNet V2 1.4 224        | TensorFlow\*                       | [mobilenet-v2-1.4-224](./mobilenet-v2-1.4-224/mobilenet-v2-1.4-224.md) | 74.09/91.97 | 1.183  | 6.087 |
+| MobileNet V3 Small 1.0      | TensorFlow\*                       | [mobilenet-v3-small-1.0-224-tf](./mobilenet-v3-small-1.0-224-tf/mobilenet-v3-small-1.0-224-tf.md) | 67.36/87.45 | 0.121 | 2.537 |
+| MobileNet V3 Large 1.0      | TensorFlow\*                       | [mobilenet-v3-large-1.0-224-tf](./mobilenet-v3-large-1.0-224-tf/mobilenet-v3-large-1.0-224-tf.md) | 75.70/92.76 | 0.4536 | 5.4721 |
```
Comment (eaidova) [inline models/public/index.md:40]: not sure, but possible open-closed-eye model should be in this list as binary classification model

File: models/public/index.md [hunk 7]
```diff
@@ -79,85 +79,85 @@ detect areas with complex shape.
...
-| RetinaFace-R50            | [MXNet\*](./retinaface-resnet50/retinaface-resnet50.md) | retinaface-resnet50 | 100.8478 | 29.427 |
-| RetinaFace-Anti-Cov     | [MXNet\*](./retinaface-anti-cov/retinaface-anti-cov.md)  | retinaface-anti-cov | 2.7781 | 0.5955 |
-| YOLO v1 Tiny              | [TensorFlow.js\*](./yolo-v1-tiny-tf/yolo-v1-tiny-tf.md) | yolo-v1-tiny-tf | 6.9883	 |	15.8587 |
-| YOLO v2 Tiny              | [TensorFlow.js\*](./yolo-v2-tiny-tf/yolo-v2-tiny-tf.md) | yolo-v2-tiny-tf | 5.4236	 |	11.2295 |
-| YOLO v2                   | [Keras\*](./yolo-v2-tf/yolo-v2-tf.md) | yolo-v2-tf | 63.0301	 |	50.9526 |
-| YOLO v3                   | [Keras\*](./yolo-v3-tf/yolo-v3-tf.md) | yolo-v3-tf | 65.9843	 |	61.9221 |
+| Model Name                           | Implementation           | OMZ Model Name | Accuracy | GFlops | mParams |
+| ------------------------------------ | ------------------------ | -------------- | -------- | ------ | ------- |
+| CTPN                                 | TensorFlow\*             | [ctpn](./ctpn/ctpn.md) | 1.3481/0.9098/1.0864 | 55.813 | 17.237 | |
```
Comment (eaidova) [inline models/public/index.md:114]: if it is percentage, then we have a trouble with accuracy...
Also it is unclear what these values mean, please stay only hmean value and remove precision, recall

File: models/public/index.md [hunk 8]
```diff
@@ -79,85 +79,85 @@ detect areas with complex shape.
...
 ### Facial Landmarks
 
-| Model Name                | Implementation | OMZ Model Name | GFlops | mParams |
-| ------------------------- | -------------- | -------------- | ------ | ------- |
-| RetinaFace-R50            | [MXNet\*](./retinaface-resnet50/retinaface-resnet50.md) | retinaface-resnet50 | 100.8478 | 29.427 |
-| RetinaFace-Anti-Cov     | [MXNet\*](./retinaface-anti-cov/retinaface-anti-cov.md)  | retinaface-anti-cov | 2.7781 | 0.5955 |
+| Model Name                | Implementation | OMZ Model Name | Accuracy | GFlops | mParams |
+| ------------------------- | -------------- | -------------- | -------- | ------ | ------- |
+| RetinaFace-R50            | MXNet\*        | [retinaface-resnet50](./retinaface-resnet50/retinaface-resnet50.md) | 87.2902 | 100.8478 | 29.427 |
+| RetinaFace-Anti-Cov       | MXNet\*        | [retinaface-anti-cov](./retinaface-anti-cov/retinaface-anti-cov.md) | 77.1531 | 2.7781 | 0.5955 |
```
Comment (eaidova) [inline models/public/index.md:149]: it is not landmarks metric value

File: models/public/index.md (hunk 5 above)
Comment (Anna-Mironova) [reply models/public/index.md]: Done

File: models/public/index.md (hunk 4 above)
Comment (Anna-Mironova) [reply models/public/index.md:50]: Fixed

File: models/public/googlenet-v1/googlenet-v1.md (hunk 3 above)
Comment (Anna-Mironova) [reply models/public/googlenet-v1/googlenet-v1.md]: Done

File: models/public/index.md (hunk 7 above)
Comment (Anna-Mironova) [reply models/public/index.md]: precision and recall have been removed

File: models/public/index.md (hunk 6 above)
Comment (Anna-Mironova) [reply models/public/index.md:40]: Added

File: models/public/index.md (hunk 8 above)
Comment (Anna-Mironova) [reply models/public/index.md]: this section has been removed

File: models/public/yolo-v3-tf/yolo-v3-tf.md [hunk 9]
```diff
@@ -53,8 +53,8 @@ Accuracy metrics obtained on COCO\* validation dataset for converted model.
 
 | Metric | Value |
 | ------ | ------|
-| mAP    | 62.27 |
 | [COCO\* mAP](http://cocodataset.org/#detection-eval) | 67.7 |
+| mAP    | 62.27 |
```
Comment (eaidova) [inline models/public/yolo-v3-tf/yolo-v3-tf.md:57]: why order changed?

File: models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md [hunk 10]
```diff
@@ -51,8 +51,8 @@ Accuracy metrics obtained on COCO\* validation dataset for converted model.
 
 | Metric | Value |
 | ------ | ------|
-| mAP    | 27.34 |
 | [COCO\* mAP](http://cocodataset.org/#detection-eval) | 29.11 |
+| mAP    | 27.34 |
```
Comment (eaidova) [inline models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md:55]: in some cases you use MAP as metric name, but here mAP, it is the same metric, could you please align?

File: models/public/face-recognition-mobilefacenet-arcface/face-recognition-mobilefacenet-arcface.md [hunk 11]
```diff
@@ -19,6 +19,10 @@ The original name of the model is [MobileFaceNet,ArcFace@ms1m-refine-v1](https:/
 
 ## Accuracy
 
+| Metric | Value |
+| ------ | ----- |
+| pairwise_accuracy_subsets | 98.8695|
```
[the remark this answers is not in the document]
Comment (Anna-Mironova) [reply models/public/face-recognition-mobilefacenet-arcface/face-recognition-mobilefacenet-arcface.md]: Done

File: models/public/googlenet-v2/googlenet-v2.md [hunk 12]
```diff
@@ -21,6 +21,11 @@ The model output for `googlenet-v2` is the typical object classifier output for
 
 ## Accuracy
 
+| Metric | Value |
+| ------ | ----- |
+| Top 1  | 72.024 |
+| Top 5  | 90.844|
+
 See [https://github.com/lim0606/caffe-googlenet-bn](https://github.com/lim0606/caffe-googlenet-bn).
```
[the remark this answers is not in the document]
Comment (Anna-Mironova) [reply models/public/googlenet-v2/googlenet-v2.md]: Done

File: models/public/yolo-v3-tf/yolo-v3-tf.md (hunk 9 above)
Comment (Anna-Mironova) [reply models/public/yolo-v3-tf/yolo-v3-tf.md]: revert

File: models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md (hunk 10 above)
Comment (Anna-Mironova) [reply models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md]: Done

**Pull request merged**